docs/commands.md

```diff
@@ -128,6 +128,20 @@ The syntactic sugar `(person_a)<-[:KNOWS]->(person_b)` will return the same resu
 
 The bracketed edge description can be omitted if all relations should be considered: `(person_a)--(person_b)`.
 
+##### Named paths
+
+Named paths are the result of assigning a matched pattern entities into a local variable. The assignment include entities that are explicitly named in the pattern, anonymous entities (without identifier) and all the entities that are implicitly part of the pattern. Named paths are objects that can be query later by designated built in functions.
```

Comment:
My recommendations:
> Named paths are the result of assigning a matched pattern entities into a local variable.

Named path variables are created by assigning a path in a MATCH clause to a single alias with the syntax:
```sh
MATCH named_path = (path)-[to]->(capture)
```

> The assignment include entities that are explicitly named in the pattern, anonymous entities (without identifier) and all the entities that are implicitly part of the pattern.

The named path includes all entities in the path, regardless of whether they have been explicitly aliased.

> Named paths are objects that can be query later by designated built in functions.

Named paths can be accessed using designated built-in functions [link to https://oss.redislabs.com/redisgraph/commands/#path-functions] or returned directly if using a language-specific client.